src/Resources/skeleton/security/Voter.tpl.php

```diff
@@ -8,15 +8,15 @@
 
 class <?= $class_name ?> extends Voter
 {
-    protected function supports($attribute, $subject)
+    protected function supports(string $attribute, $subject)
```

Comment:
The type-hint isn't added to Symfony until 5.0. I can never remember the rules on this, but it seems like a user cannot *add* the type on their class if the parent doesn't have it: https://3v4l.org/uUIYl - but I can't remember if this is the whole story...

Reply:
This also makes me wonder if we should at least test the LTS Symfony version in our test matrix...